packages/steveo/src/lib/manager.ts

```diff
@@ -53,11 +87,12 @@ export class Manager {
         this.forceTerminate();
         break;
       }
-      this.steveo.logger.debug(`waiting for consumers to terminate`);
+      this.steveo.logger.debug('waiting for consumers to terminate');
```

Comment:
Same, it might be nice to see the log

```suggestion
      this.steveo.logger.info('waiting for consumers to terminate');
```

Reply:
I am wary of the volume log. That log would appear every seconds while it's trying to shutdown every time.

Reply:
Oh right, I thought this executed on Steveo shutdown only